test/java/seedu/addressbook/data/AddressBookTest.java

```diff
@@ -75,11 +75,10 @@ public void setUp() throws Exception {
 
     @Rule
     public ExpectedException thrown = ExpectedException.none();
-    
 
     @Test
     public void addPerson_emptyAddressBook() throws Exception {
-        emptyAddressBook.addPerson(bobChaplin); 
+        emptyAddressBook.addPerson(bobChaplin);
```

Comment:
Avoid auto-formatting unrelated parts of the code. Doing so can result in unrelated changes being included in the commit. You can use sourcetree to look through changes before committing, and discard any unrelated changes. If you wish to commit formatting changes to unrelated parts of the code, do so in a separate commit.